local-k8s.sh

```diff
@@ -283,9 +291,7 @@ function clean() {
         helm uninstall weaviate -n weaviate
     fi
 
-    if [[ $need_minio == "true" ]]; then
-        shutdown_minio
-    fi
+    shutdown_minio
```

Comment:
Was this intentinally? I believe we still need the condition `    if [[ $need_minio == "true" ]]; then` , don't we?